assets/css/style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #d9dcd6;
+}
+
+.header {
+  padding: 20px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+    "Lucida Sans", Arial, sans-serif;
+  background-color: #2a607c;
+  color: #ffffff;
+}
+
+.header h1 {
+  display: inline-block;
+  font-size: 48px;
+}
+
+.header h1 .seo {
+  color: #d9dcd6;
+}
+
+.header nav {
+  padding-top: 15px;
+  margin-right: 20px;
+  float: right;
+  font-family: "Gill Sans", "Gill Sans MT", "Calibre", "Trebuchet MS",
+    sans-serif;
+  font-size: 20px;
+}
+
+.header nav ul li {
+  display: inline-block;
+  margin-left: 25px;
+  list-style-type: none;
+}
+
+a {
+  color: #ffffff;
+  text-decoration: none;
+}
+
+p {
+  font-size: 16px;
+}
+
+.hero {
+  height: 800px;
+  width: 100%;
+  margin-bottom: 25px;
+  background-image: url("../images/digital-marketing-meeting.jpg");
+  background-size: cover;
+  background-position: center;
+}
+
+.float-left {
+  float: left;
+  margin-right: 25px;
+}
+
+.float-right {
+  float: right;
+  margin-left: 25px;
+}
+
+.content {
+  width: 75%;
+  display: inline-block;
+  margin-left: 20px;
+}
+
+.benefits {
+  margin-right: 20px;
+  padding: 20px;
+  clear: both;
+  float: right;
+  width: 20%;
+  height: 100%;
+  font-family: "Gill Sans", "Gill Sans MT", "Calibre", "Trebuchet MS",
+    sans-serif;
+  background-color: #2589bd;
+}
+
+.benefit {
+  margin-bottom: 32px;
+  color: #ffffff;
+}
+
+.benefit,
+h3 {
+  margin-bottom: 10px;
+  text-align: center;
+}
+
+.benefit img {
+  display: block;
+  margin: 10px auto;
+  max-width: 150px;
+}
+
+/*main 3 sections where images are*/
+
+.search-engine-optimization {
+  margin-bottom: 20px;
+  padding: 50px;
+  height: 300px;
+  font-family: "Gill Sans", "Gill Sans MT", "Calibre", "Trebuchet MS",
+    sans-serif;
+  background-color: #0072bb;
+  color: #ffffff;
+}
+
+/*main 3 images of the sections*/
+
+.search-engine-optimization img {
+  max-height: 200px;
+}
+
+/*
+
+.social-media-marketing img {
+  max-height: 200px;
+}
```

Comment:
If code is no longer used, delete it. Commenting out is fine when you're testing but delete before pushing.